docs/features/backup_and_restore.md

```diff
@@ -0,0 +1,125 @@
+
+#### Restoring an individual table
+
+A table can be restored to a previous state using PITR. This is easily achievable via the AWS Console by visiting the `Backups` tab for the table.
+
+A table can only be recovered to a new table name. Following the restoration of the table, the new table must be imported into Terraform.
+
+First, remove the old table from the Terraform state:
+
+```sh
+terraform state rm module.data_persistence.aws_dynamodb_table.collections_table
+```
+replacing `collections_table` with the table identifier in the [DynamoDB Terraform table definitions](https://github.com/nasa/cumulus/blob/master/tf-modules/data-persistence/dynamo.tf).
+
+Then import the new table into the Terraform state:
+
+```sh
+terraform import module.data_persistence.aws_dynamodb_table.collections_table <new-table-name>
+```
+
+replacing `collections_table` with the table identifier in the [DynamoDB Terraform table definitions](https://github.com/nasa/cumulus/blob/master/tf-modules/data-persistence/dynamo.tf).
+
+Your `data-persistence` and `cumulus` deployments should be redeployed so that your instance of Cumulus uses this new table. After the deployment, your Elasticsearch instance will be out of sync with your new table if there is any change in data. To resync your Elasticsearch with your database run an [`index-from-database` operation](https://nasa.github.io/cumulus-api/#index-from-database) via the Operator API. The status of this operation can be viewed on the dashboard. When Elasticsearch is switched to the new index the DynamoDB tables and Elasticsearch instance will be in sync and the correct data will be reflected on the dashboard.
+
+### Backup and Restore with cumulus-api CLI
+
+cumulus-api CLI also includes a backup and restore command. The CLI backup command downloads the content of any of your DynamoDB tables to `.json` files. You can also use these `.json` files to restore the records to another DynamoDB table.
+
+#### Backup with the CLI
+
+To backup a table with the CLI, install the `@cumulus/api` package using [npm](https://www.npmjs.com/), making sure to install the same version as your Cumulus deployment:
+
+```bash
+npm install -g @cumulus/api@version
+```
+
+Then run:
+
+```bash
+cumulus-api backup --table <table-name>
+```
+
+the backup will be stored at `backups/<table-name>.json`
+
+#### Restore with the CLI
+
+To restore data from a json file run the following command:
+
+```bash
+cumulus-api restore backups/<table-name>.json --table <new-table-name>
```

Comment:
So you said this doesn't need to be a new table right? If so, it should just say `<table-name>`